pc/request.hpp

```diff
@@ -243,6 +247,12 @@ namespace pc
     // convenience wrapper equiv to: get_product()->get_symbol()
     str get_symbol();
 
+    // Get the base currency for the product (from the list of product attributes).
+    str get_base();
```

Comment:
This is not the base currency but the base asset, rename `get_base_asset` (might not be a currency)